Clients/GUI/ViewModels/UserInfoViewModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace GUI.ViewModels
+{
+    public class UserInfoViewModel
```

Comment:
When exactly is this model shown up? Why do we require name, surname and DateOfBirth?

Reply:
> When exactly is this model shown up? Why do we require name, surname and DateOfBirth?

When: I described that in one of the previous replies.
Why: because we had such a task description in trello

Reply:
Can't we check only one of them?

Reply:
> Can't we check only one of them?

Only one of the properties or what?